tmobile/jazz
# Test Lambda Sidebar

Environment Try Service Sidebar Component

UI implementation for the in-app lamda tester

adds kotlin.js (1497kb) and really-relaxed-json.js (61kb)

steps.
1. Create Lamda
2. Go to environments (prod will do)
3. Click [TEST ARN] to open sidemenu
4. Write poorly formatted json in input text box
5. Click [FORMAT] to format json
6. Click [TEST] to see stubbed results in output
